styles/globals.js

```diff
@@ -19,13 +19,22 @@ export const markdownComponents = {
     <a className="text-purple-800 hover:text-blue-800" {...props} />
   ),
   // Apply margin to paragraphs to create space between them
-  p: ({ node, ...props }) => <p className="mb-4" {...props} />,
+  p: ({ node, ...props }) => (
+    <>
+      <p className="inline" {...props} />
+      <br />
+    </>
+  ),
+  strong: ({ node, ...props }) => <strong {...props}></strong>,
   // Apply tailwind classes to style lists
   ul: ({ node, ...props }) => (
-    <ul className="list-disc list-inside ml-4" {...props} />
+    <>
+      <ul className="list-disc list-inside ml-4 inline-block" {...props} />
+      <br />
```

Comment:
what are we trying to solve with this? I wonder, how other lists will look, it's a bit risky to update global styles to fix something on one page

Reply:
There was an issue with list items appearing as:
*
Content
Instead of:
* Content

This fixes the issue without affecting other <ul> components on the site as <Markdown> tag is only used on the blog posts and education articles:
![image](https://github.com/user-attachments/assets/13f75853-0eb6-472a-8935-3f73a9c18b89)